CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 2.8.5)
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
Could I ask you to check `grep std=c++11 * -R` in a build directory without this line?